lib/config/size.go

```diff
@@ -78,7 +78,7 @@ func (s *Size) ParseDefault(str string) error {
 	return err
 }
 
-func checkFreeSpace(req Size, usage fs.Usage) error {
+func CheckFreeSpace(req Size, usage fs.Usage) error {
```

Comment:
Move to osutil or fsutil or something?

Reply:
I guess size would have to move too

Reply:
Indeed. I was thinking about moving the entire size thing to `fs`, as `Usage` lives there and juggling bytes seems more appropriate for `fs` than `config`, but didn't in favor of not moving code around. Do you consider `fs` as suitable?

Reply:
It's a bit weird as size is this auxilary struct that does not represent anything in fs in general.